src/TrackerSnapshot.php

```diff
@@ -57,8 +56,18 @@ protected static function include_snapshot_data( array $data = array() ): array
 			$data['extensions'] = array();
 		}
 
+		$feed_generation_time = ProductFeedStatus::get()[ ProductFeedStatus::PROP_FEED_GENERATION_WALL_TIME ];
+
 		$data['extensions'][ PINTEREST_FOR_WOOCOMMERCE_TRACKER_PREFIX ] = array(
 			'settings' => self::parse_settings(),
+			'store'    => array(
+				'connected'        => wc_bool_to_string( Pinterest_For_Woocommerce::is_connected() ),
+				'actively_syncing' => wc_bool_to_string( ProductSync::is_product_sync_enabled() ),
```

Comment:
This will be included in the `settings` section. It has `product_sync_enabled` prop tracked.
It would be better to track if the feed is registered. You can use `Pinterest_For_Woocommerce()::get_data( 'feed_registered' )` but don't store the ID just check if it is not null.